src/GUI/UI/UIGameOver.cpp

```diff
@@ -0,0 +1,56 @@
+#include "Engine/Engine.h"
+#include "Engine/EngineGlobals.h"
+#include "Engine/Localization.h"
+#include "Engine/Time.h"
+#include "Engine/Graphics/IRender.h"
+
+#include "GUI/UI/UIGameOver.h"
+#include "Application/GameOver.h"
+
+GUIWindow_GameOver::GUIWindow_GameOver(UIMessageType on_release_event)
+    : GUIWindow(WINDOW_GameOverWindow, {0, 0}, render->GetRenderDimensions(), on_release_event) {
+    pEventTimer->Pause();
+    prev_screen_type = current_screen_type;
+    current_screen_type = CURRENT_SCREEN::SCREEN_GAMEOVER_WINDOW;
+    _tickcount = platform->tickCount() + 5000;
```

Comment:
Just 5s to read the certificate? Or am I missing something?

Reply:
5 second delay before message box comes up. In GOG version its unlimited wait till esc press. But we had the delay in the code already so i kept it.